### PR TITLE
Migration init CM Table

### PR DESCRIPTION
**Pull request by soupette**
### What does it do?

It starts implementing the table in the CM.

### Why is it needed?

Describe the issue you are solving.

### How to test it?

Provide information about the environment and the path to verify the behaviour.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Users/ListPage/PaginationFooter/PageSize.js [hunk 1]
```diff
@@ -20,20 +20,27 @@ const PageSize = () => {
...
+        onChange={handleChange}
+        value={pageSize}
+      >
         <Option value="10">10</Option>
         <Option value="20">20</Option>
         <Option value="50">50</Option>
         <Option value="100">100</Option>
       </Select>
-      <Box paddingLeft={2}>
+      {/* <Box paddingLeft={2}>
```
Comment (mfrachet) [inline packages/core/admin/admin/src/pages/SettingsPage/pages/Users/ListPage/PaginationFooter/PageSize.js:36]: Still needed?

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Users/ListPage/utils/api.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import { axiosInstance } from '../../../../../../core/utils';
 
-const fetchData = async search => {
+const fetchData = async (search, notify) => {
```
Comment (mfrachet) [inline packages/core/admin/admin/src/pages/SettingsPage/pages/Users/ListPage/utils/api.js:3]: 🙏🏻

**Review by mfrachet — APPROVED** [review]
LGTM, just mentionning a comment. Good job 🙏🏻

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Users/ListPage/PaginationFooter/PageSize.js (hunk 1 above)
Comment (soupette) [reply packages/core/admin/admin/src/pages/SettingsPage/pages/Users/ListPage/PaginationFooter/PageSize.js:36]: Let's leave it if we are asked to match the design of the page size footer in the select.

**Pull request merged**